lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -642,6 +642,31 @@ sub _add_distri_and_version_to_summary {
     }
 }
 
+sub latest_jobs {
+    my ($self, $search_args, $until) = @_;
```

Comment:
When adding new functions or changing arguments it would be nice to make use of signatures.